Resources/template.json

```diff
@@ -0,0 +1,297 @@
+        }
+      },
+      "ApimServiceName": {
+        "type": "string",
+        "metadata": {
+          "description": "APIM name"
+        }
+      },
+      "ApimResourceGroup": {
+        "type": "string",
+        "metadata": {
+          "description": "Resource group where the APIM is located"
+        }
+      },
+      "ApiName": {
+        "type": "string",
+        "metadata": {
+          "description": "API name"
+        }
+      },
+      "apiSuffix": {
+        "type": "string",
+        "defaultValue": "",
+        "metadata": {
+          "description": "Suffix uniquely identifying this API and all of its resource paths within APIM, defaults to apiName"
+        }
+      },
+      "cosmosDBfqdn": {
+        "type": "string",
+        "metadata": {
+          "description": "Cosmos DB account name in the format https://dfc-xxx-xxx-cdb.documents.azure.com/ for app settings"
+        }
+      },
+      "cosmosDBkey": {
+        "type": "securestring",
+        "metadata": {
+          "description": "Cosmos DB access key for app settings"
+        }
+      },
+      "cosmosDBdatabase": {
+        "type": "string",
+        "metadata": {
+          "description": "Cosmos DB database name for app settings"
+        }
+      },
+      "cosmosDbCollectionFeChoices": {
+        "type": "string",
+        "metadata": {
+          "description": "Cosmos DB collection name for app settings"
+        }
+      },
+      "cosmosDbCollectionFrameworks": {
+        "type": "string",
+        "metadata": {
+          "description": "Cosmos DB collection name for app settings"
+        }
+      },
+      "cosmosDbCollectionProgTypes": {
+        "type": "string",
+        "metadata": {
+          "description": "Cosmos DB collection name for app settings"
+        }
+      },
+      "cosmosDbCollectionSsaTier1s": {
+        "type": "string",
+        "metadata": {
+          "description": "Cosmos DB collection name for app settings"
+        }
+      },
+      "cosmosDbCollectionSsaTier2s": {
+        "type": "string",
+        "metadata": {
+          "description": "Cosmos DB collection name for app settings"
+        }
+      },
+      "cosmosDbCollectionStandards": {
+        "type": "string",
+        "metadata": {
+          "description": "Cosmos DB collection name for app settings"
+        }
+      },
+      "cosmosDbCollectionStandardsSectorCodes": {
+        "type": "string",
+        "metadata": {
+          "description": "Cosmos DB collection name for app settings"
+        }
+      }
+    },
+    "variables": {
+      "deploymentUrlBase": "https://raw.githubusercontent.com/SkillsFundingAgency/dfc-devops/master/ArmTemplates/",
+      "functionAppName": "[concat(parameters('functionAppPrefix'),'-fa')]",
+      "appInsightName": "[concat(parameters('functionAppPrefix'),'-ai')]"
+    },
+    "resources": [{
+        "apiVersion": "2017-05-10",
+        "name": "pprefdataAppInsights",
```

Comment:
Looks like this was copy and pasted from another project without updating to reflect the name of this project